src/commands/force/source/beta/push.ts

```diff
@@ -89,13 +91,19 @@ export default class Push extends DeployCommand {
 
     // fire predeploy event for sync and async deploys
     await this.lifecycle.emit('predeploy', componentSet.toArray());
-    this.ux.log(`*** Pushing with ${this.isRest ? 'REST' : 'SOAP'} API v${componentSet.sourceApiVersion} ***`);
+
+    const isRest = await this.isRestDeploy();
+    this.ux.log(
+      `*** Pushing with ${isRest ? 'REST' : 'SOAP'}${
+        componentSet.sourceApiVersion ? ` API v${componentSet.sourceApiVersion}` : ''
+      } ***`
+    );
```

Comment:
I thought you were going to report both sourceApiVersion (when different from apiVersion) and apiVersion?  If you're going to report 1, then I would choose apiVersion.  Imagine if someone is overriding the apiVersion on the command line or by env var and the command reports the sourceApiVersion.  That would be confusing.

Reply:
OK, here's that for push and deploy.